sql-plugin/src/main/scala-2.13/com/nvidia/spark/rapids/ArmScalaFixes.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2020-2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nvidia.spark.rapids
+
+import scala.collection.{immutable, mutable}
+import scala.collection.mutable.ListBuffer
+import scala.util.control.ControlThrowable
+
+import com.nvidia.spark.rapids.RapidsPluginImplicits._
+
+/** Implementation of the automatic-resource-management pattern */
+trait ArmScalaFixes {
```

Comment:
maybe a nit but I assume this is just scala specific implementation details, rename ArmScalaSpecificImpls or ArmScalaVersionImpls